reducer/src/main/java/com/graphicsfuzz/reducer/ReductionDriver.java

```diff
@@ -282,7 +282,9 @@ public String doReduction(
       while (true) {
         LOGGER.info("Trying reduction attempt " + stepCount + " (" + numSuccessfulReductions
             + " successful so far).");
-        final Optional<ShaderJob> maybeNewState = passManager.applyReduction(currentState);
+        final Optional<ShaderJob> maybeNewState;
+        maybeNewState = passManager.applyReduction(currentState);
+
```

Comment:
Can you revert this to what it was previously?  No need to split this into multiple lines.